frontend/src/features/mapSideBar/tabs/Section.tsx

```diff
@@ -23,31 +19,47 @@ export const Section: React.FC<SectionProps> = ({
   const [isCollapsed, setIsCollapsed] = useState<boolean>(!initiallyExpanded && true);
   return (
     <StyledFormSection>
-      <StyledSectionHeader>
-        <Row>
-          <Col>{header}</Col>
-          <Col xs="1">
-            {isCollapsable && isCollapsed && (
-              <ArrowDropDownIcon
-                onClick={() => {
-                  setIsCollapsed(!isCollapsed);
-                }}
-              />
-            )}
-            {isCollapsable && !isCollapsed && (
-              <ArrowDropUpIcon
-                onClick={() => {
-                  setIsCollapsed(!isCollapsed);
-                }}
-              />
-            )}
-          </Col>
-        </Row>
-      </StyledSectionHeader>
+      {header && (
```

Comment:
Allows the section to be without a header if necessary. Allows the use of this component instead of only the style

Reply:
Also I think this component should be moved to common folder. We are using it in most forms

Reply:
Agreed. Can you log a tech debt to do this exclusively?